portfolio/src/main/webapp/script.js

```diff
@@ -124,4 +134,18 @@ function uploadComment() {
       // TODO: handle case when verification fails
     }
   })
-}
\ No newline at end of file
+}
+
+function deleteComment(id) {
+  fetch('/delete-comment', {
+    method: 'POST',
+    body: "id=" + id,
+    headers: {
+      "Content-Type": "application/x-www-form-urlencoded"
+    }
+  }).then(function(response) {
+    if (!response.ok) {
+      // failure
```

Comment:
`console.error()` would be appropriate here if you don't show anything in the ui.